crates/nargo_cli/tests/test_data_ssa_refactor/brillig_oracle/src/main.nr

```diff
@@ -0,0 +1,15 @@
+// Tests oracle usage in brillig/unconstrained functions
+fn main(x: Field)  {
+    // call through a brillig wrapper
+    oracle_print_wrapper(x);
+}
+
+
+#[oracle(oracle_print_impl)]
+unconstrained fn oracle_print(x : Field) -> Field {}
```

Comment:
We are either going to have to intern this function in the resolver (which breaks away from foreign funcs and builtins) or prefix all inputs to oracle funcs with "_". I'm leaning towards just interning the func as we have long wanted to deprecate "_"

Reply:
Can you explain more about 'breaking away from foreign funcs by interning in the resolver'?

Reply:
I meant here in the resolver (https://github.com/noir-lang/noir/blob/537d33dfdfa6195909af44dc919f8f0a15b57cf3/crates/noirc_frontend/src/hir/resolution/resolver.rs#L306). I just pushed [this PR](https://github.com/noir-lang/noir/pull/1612) though and chose to require inputs to oracle to be prefixed with an `_` for now.